source/blueprints/profile/routes.py

```diff
@@ -10,3 +18,28 @@
 def profile_index():
     """Index request."""
     return jsonify({"data": "Profile"})
+
+
+@profile_bp.route("/reports", methods=["POST"])
```

Comment:
This code is correct, but it would be better to create a new blueprint to hold all of the `reports` logic instead of using the `profile` blueprint.

Check how I'm creating the blueprints in the `source/blueprints/*.py` files and follow the same directory/file structure.

Also, once you create the `report` blueprint, remember to register it in the `source/__init__.py` file under the `register_blueprints()` function.